plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2761,26 +2672,7 @@ public void testAllAvailableTypes()
                 "  data.a_uuid " +
                 " FROM \"test_all_types$partitions\" "))
                 .matches(
-                        format == ORC
-                                ? "VALUES (" +
-                                "  BIGINT '2', " +
-                                "  BIGINT '2', " +
-                                "  CAST(NULL AS ROW(min boolean, max boolean, null_count bigint)), " +
-                                "  CAST(NULL AS ROW(min integer, max integer, null_count bigint)), " +
-                                "  CAST(NULL AS ROW(min bigint, max bigint, null_count bigint)), " +
-                                "  CAST(NULL AS ROW(min real, max real, null_count bigint)), " +
-                                "  CAST(NULL AS ROW(min double, max double, null_count bigint)), " +
```

Comment:
where did these go to?

Reply:
Before my changes the ORC version of this table was deemed to have invalid column metrics, thus the NULL min/max/null count rows. I think that was a bug though, and the stats are the same now for both Parquet and ORC, with a few exceptions below.